### ISSUE TITLE
Problem with the release Github workflow order

### ISSUE THREAD

**Issue by sp-ricard-valverde** [socialpoint-labs/spine-json-lib#40]
* spine-json-lib version:
* Python version:
* Operating System:

### Description

The Github Workflow order does not guarantee a broken version to be published to Pypi.

### What I Did

When a PR is merged, the workflows for Test and Release are run with the PR changes merged into the `master` branch.

But they are ran in parallel so in the case of the Tests workflow failing in the `master` branch, the package would still be published.
